lib/chef/knife/bootstrap.rb

```diff
@@ -614,9 +614,9 @@ def perform_bootstrap(remote_bootstrap_script_path)
         end
       end
 
-      def connect!
+      def connect!(opts = nil)
```

Comment:
Any thoughts to [Keyword Arguments](https://thoughtbot.com/blog/ruby-2-keyword-arguments) ?